resources/authgear/templates/en/web/authflowv2/__password_input.html

```diff
@@ -0,0 +1,55 @@
+{{ define "authflowv2/__password_input.html" }}
+<div class="{{ $.Classname }} relative" data-controller="password-visibility-toggle">
+	<input
+		tabindex="0"
+		class="input input__password w-full"
+		{{ if .AutoFocus }}autofocus{{ end }}
+		{{ if eq .Type "new-password" }}
+		data-password-policy-target="input"
+		data-action="password-policy#check"
+		{{ end }}
+		{{ if .AutoFocus }}autofocus{{ end }}
+		type="password"
+		{{ if eq .Type "old-password" }}
+		autocomplete="current-password"
+		{{ else if eq .Type "new-password" }}
+		autocomplete="new-password"
+		{{ end }}
+		autocapitalize="none"
+		name="{{ .Name }}"
+		{{ if eq .Type "old-password" }}
+		placeholder="{{ template "v2-password-placeholder" }}"
+		{{ else if eq .Type "new-password" }}
+		placeholder="{{ template "v2-new-password-placeholder" }}"
+		{{ else if eq .Type "confirm-password" }}
+		placeholder="{{ template "v2-confirm-password-placeholder" }}"
+		{{ end }}
+		data-password-visibility-toggle-target="input"
+		{{ if .PasswordRules }}
+		passwordrules="{{ .PasswordRules }}"
+		{{ end }}
+	>
+	<button
+		class="w-5 absolute inset-y-0 right-4"
+		type="button"
+		tabindex="-1"
+		title="{{ template "show-password-label" }}"
+		data-password-visibility-toggle-target="showButton"
+		data-action="password-visibility-toggle#show">
+    <span class="material-icons align-middle text-xl">
+      visibility_off
+    </span>
+	</button>
+	<button
+		class="w-5 absolute inset-y-0 right-4 hidden"
+		type="button"
+		tabindex="-1"
+		title="{{ template "hide-password-label" }}"
+		data-password-visibility-toggle-target="hideButton"
+		data-action="password-visibility-toggle#hide">
+      <span class="material-icons align-middle text-xl">
+        visibility
+      </span>
+	</button>
```

Comment:
The indentation here is a bit weird. (tab / space?)
Maybe convert all to tabs.

Reply:
I think this should be done by a tool, and be enforced on CI (we currently do not have it yet).

Reply:
This website uses Go template https://github.com/golang/pkgsite I did not find any configuration related to formatting the go templates :(